### ISSUE TITLE
`wayland-client.h no such file or directory` but package `wayland-devel` is installed

### ISSUE THREAD

**Issue by ElijaRock** [xyproto/wallutils#38]
I am on OpenSUSE Tumbleweed with `wayland-devel` package installed. `ls /usr/lib | grep wayland` shows `libwayland-client.so` however I still get the following compilation error with `make`:

```
$ make
go build -mod=vendor -trimpath
# github.com/xyproto/wallutils
In file included from ./wayinfo.go:4:
./include/wayinfo.h:32:10: fatal error: wayland-client.h: No such file or directory
   32 | #include <wayland-client.h>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:18: all] Error 1
```
I have tried installing `qt6-waylandclient-devel`, `wayland-devel-32bit`, `xwayland-devel`, `libwayland-client0`, `libQt5WaylandClient5`, `libQt6WaylandClient6`, and `kwayland`, but nothing has fixed the `make` error.

**Comment by xyproto** [comment]
Thanks for reporting! It looks like the file locations have changed for OpenSUSE Tumbleweed.

I'll try building it in OpenSUSE Tumbleweed myself.

I might also add release binaries and/or switch to using a Wayland implementation that is in pure Go.

**Comment by xyproto** [comment]
Fixed in #39
